templates/zerver/index.html

```diff
@@ -128,4 +128,6 @@
     {% include "zerver/tutorial_finale.html" %}
     <div class='notifications top-right'></div>
 </div>
+<!--This is just to retrieve the sprite image from the server and cache it-->
+<div class="emoji emoji-100" style='margin-left: -1000'></div>
```

Comment:
Hmm.  Is there a way we can do this so that the network traffic is deferred until after the page is loaded, and it doesn't take up space in the DOM?